setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from setuptools import find_packages, setup
+
+with open("README.md", "r") as readme_file:
+    readme = readme_file.read()
+
+with open("requirements.txt", "r") as reqs_file:
+    requirements = reqs_file.readlines()
+
+setup(
+    name="gitlab-configuration-as-code",
+    version="1.0.0",
```

Comment:
If you version the package anyway with git you might want to look at: https://pypi.org/project/setuptools-scm/
We use it at some projects and I realy like that you only have to maintain the versioning with git tags.

Reply:
I have created issue to do this ;)